### PR TITLE
Huge code refactor + style cop config

### PR DESCRIPTION
**Pull request by AndriiS1**

### CODE REVIEW

File: ASPNET-CORE-MVC/Domain/Dtos/TransactionDtos/TransactionDto.cs [hunk 1]
```diff
@@ -21,8 +21,7 @@ public TransactionDto(Transaction transaction)
         Planned = transaction.Planned;
     }
 
-    [Required]
-    public Guid Id { get; set; }
+    [Required] public Guid Id { get; set; }
```
Comment (HoshkoS) [inline ASPNET-CORE-MVC/Domain/Dtos/TransactionDtos/TransactionDto.cs:24]: Could it be `public required` as in other DTOs?

File: ASPNET-CORE-MVC/ThinkTwice/Controllers/AuthController.cs [hunk 2]
```diff
@@ -1,230 +1,189 @@
...
         }
-        catch (Exception e)
+
+
+        [HttpPost]
+        public async Task<ActionResult> CreateUser(RegisterUserDto user)
         {
-            _flashMessage.Danger(e.Message);
-            return RedirectToAction("SignUp", "Auth");
+            try
```
Comment (HoshkoS) [inline ASPNET-CORE-MVC/ThinkTwice/Controllers/AuthController.cs]: Could you please remove `try... catch` from controller methods

File: ASPNET-CORE-MVC/Domain/Dtos/TransactionDtos/TransactionDto.cs (hunk 1 above)
Comment (AndriiS1) [reply ASPNET-CORE-MVC/Domain/Dtos/TransactionDtos/TransactionDto.cs:24]: This is different costructions. Keyword "required" means that the filed must be set during initialiation, [Required] is about that field that comes from form must be set.

**Review by wehfis — APPROVED** [review]
perfect

**Pull request merged**